### PR TITLE
Update train_and_evaluate_using_ray.md

### PR DESCRIPTION
**Pull request by Grahlus**
evaluation.csv was not found because the actual directory changed during the run.
-> Added Get the current working directory and add it to csv_filename.

### CODE REVIEW

File: docs/source/examples/train_and_evaluate_using_ray.md [hunk 1]
```diff
@@ -171,12 +175,12 @@ env_config_training = {
     "window_size": 14,  # We want to look at the last 14 samples (hours)
     "reward_window_size": 7,  # And calculate reward based on the actions taken in the next 7 hours
     "max_allowed_loss": 0.10,  # If it goes past 10% loss during the iteration, we don't want to waste time on a "loser".
-    "csv_filename": "training.csv"  # The variable that will be used to differentiate training and validation datasets
+    "csv_filename": cwd + "/training.csv"  # The variable that will be used to differentiate training and validation datasets
```
Comment (carlogrisetti) [inline docs/source/examples/train_and_evaluate_using_ray.md:178]: you should use the cross-platform way of doing this, with:
os.path.join(cwd, 'training.csv')

File: docs/source/examples/train_and_evaluate_using_ray.md [hunk 2]
```diff
@@ -171,12 +175,12 @@ env_config_training = {
...
 }
 # Specific configuration keys that will be used during evaluation (only the overridden ones)
 env_config_evaluation = {
     "max_allowed_loss": 1.00,  # During validation runs we want to see how bad it would go. Even up to 100% loss.
-    "csv_filename": "evaluation.csv",  # The variable that will be used to differentiate training and validation datasets
+    "csv_filename": cwd + "/evaluation.csv",  # The variable that will be used to differentiate training and validation datasets
```
Comment (carlogrisetti) [inline docs/source/examples/train_and_evaluate_using_ray.md:183]: you should use the cross-platform way of doing this, with:
os.path.join(cwd, 'evaluation.csv')

**Review by carlogrisetti — CHANGES_REQUESTED** [review]
Please switch to os.path.join instead of concatenating strings

**Comment by Grahlus** [comment]
Changed to os.path.join

**Comment by carlogrisetti** [comment]
Thanks!

**Pull request merged**